### ISSUE TITLE
Suggestion: /altergen command

### ISSUE THREAD

**Issue by varkarrus** [thadunge2/AIDungeon#40]
I wrote this command the other day, as a variant of my /alter command, which is in this fork.

```
            elif command == "altergen": 
                if len(story_manager.story.results) is 0: 
                    console_print("There's no results to alter.\n") 
                    continue 
                console_print("\nThe AI thinks this was what happened:\n") 
                print(story_manager.story.results[-1])
                action = story_manager.story.actions[-1]
                story_manager.story.actions = story_manager.story.actions[:-1]
                story_manager.story.results = story_manager.story.results[:-1] #most recent action and result needs to be temporarily removed to let storycontext work
                result = input("\nWrite the first part of the new result (use \\n for new line):\n\n") 
                result = result.replace("\\n", "\n")
                result += story_manager.generator.generate(story_manager.story_context() + "\n> "+action +"\n" + result)
                print(result)
                story_manager.story.add_to_story(action,result)
```

**Comment by thadunge2** [comment]
Feel free to open a PR! Seems like an interesting feature.

**Comment by varkarrus** [comment]
To be honest, I don't know how to do that :|

(sorry)

**Comment by ShnitzelKiller** [comment]
Fork this repository, then for each feature you want to make a PR request for, make a new branch. When you push a new branch, Github will even detect it and ask if you want to make a pull request using that branch in the Pull Requests tab.

**Comment by marktaiwan** [comment]
This feature is definitely nice to have, I'll work on it.

**Comment by thadunge2** [comment]
Closed by #81
